sdk/include/psdk/uxtheme.h

```diff
@@ -192,6 +216,25 @@ DrawThemeTextEx(
     _Inout_ LPRECT pRect,
     _In_ const DTTOPTS *options
 );
+
+// vista+
```

Comment:
```suggestion
// Vista+
```
with compile time version check?

Reply:
No.